int64_test.go

```diff
@@ -22,8 +22,10 @@ package atomic
 
 import (
 	"encoding/json"
+	"math"
```

Comment:
i think we forgot to mark the `_test.go` file as generated, since it's not hidden by default

Reply:
Test files aren't generated. These are all hand-written. We're generating the
code, but the tests are for humans to verify the behavior.